main.py

```diff
@@ -290,16 +291,17 @@ def make(self):
         fruitscore = 0
         score = 0
         
-        if os.path.exists("score.pkl")==False:
-            open('score.pkl','w+')
+        score_path = get_activity_root() + '/instance' + '/score.pkl'
```

Comment:
Please use `os.path.join` to join file paths

Reply:
Fixed it @pro-panda